diff -Naur a/src/nvidia/src/kernel/gpu/mem_mgr/mem_mgr.c b/src/nvidia/src/kernel/gpu/mem_mgr/mem_mgr.c
--- a/src/nvidia/src/kernel/gpu/mem_mgr/mem_mgr.c	2026-07-18 11:26:26.862962399 -0700
+++ b/src/nvidia/src/kernel/gpu/mem_mgr/mem_mgr.c	2026-08-06 20:45:00.000000000 +0900
@@ -4138,7 +4138,9 @@

     if (!bFifoLite && !RMCFG_FEATURE_PLATFORM_GSP && bVirtualMode &&
         IS_SILICON(pGpu) && !pMemorySystemConfig->bDisableCompbitBacking &&
-        (pMemorySystemConfig->bUseRawModeComptaglineAllocation || pMemorySystemConfig->bOneToOneComptagLineAllocation))
+        (pMemorySystemConfig->bUseRawModeComptaglineAllocation || pMemorySystemConfig->bOneToOneComptagLineAllocation) &&
+        ((pGpu->idInfo.PCIDeviceID >> 16) != 0x20C2 &&
+         (pGpu->idInfo.PCIDeviceID >> 16) != 0x2082))
     {
         // Turn on virtual mode to enable compressed allocation access
         ceUtilsParams.flags |= DRF_DEF(0050_CEUTILS, _FLAGS, _VIRTUAL_MODE, _TRUE);
